### ISSUE TITLE
Slim sidebar issues

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#8311]
Compiling a list of what people have shared here and there. (https://github.com/wagtail/wagtail/pull/8197#issuecomment-1087493676, internal discussions)

- After RC:
- [x] **Visual design** – Collapsed sidebar is wider than intended
- [x] **Visual design** – Change “Edit your account” to “Account” tooltip
- [x] **Visual design** – Slim side bar on bird hover, make the bird a little big bigger (don’t make it shrink as much)
- [x] **Visual design** – When the sidebar loads, the area underneath is grey. Could we switch this to match the darker hue of the sidebar’s background?
- [x] **Visual design** – Change the tooltip to the darker indigo
- [x] **Visual design** – Expand the space between the menu items (add 3px padding) - then adjust the flyout menu items spacing too
- [x] **Visual design** – Get the labels and icons to align vertically (space between icon and label)
- [x] **Visual design** – Make the width 60px
- [x] **Visual design** – Make the arrows slightly smaller and closer to the icons on slim view
- [x] **Visual design** – Move the bird higher up and slightly smaller, as per the designs
- [x] **Animations** – The close animation for sub-menus doesn't seem to play when the menu is expanded
- [x] **Animations** – The account menu seems to have a different tween animation to the rest of the menu. Causing it to do a weird thing when you collapse the menu while the account menu is open
- [x] **Animations** – The avatar suddenly jumps to the right when you collapse the menu
- [x] **Animations** – I wonder if the new `|->` icon should flip horizontally rather than roll? (or maybe not animate at all?) The rolling animation feels a little strange as I think the pipe is meant to represent the always vertical menu
- [x] **Animations** – The Bird seems to have two hover states (try slowly moving your mouse cursor from top to bottom and you'll see it's wing appears before the hover animation is triggered). Not sure if this is intended behaviour.
- [ ] **Accessibility** – Unexpected tab order #8035 
- ~~[ ] **UX** – Unexpected behavior with the search – clicking the "Search" icon submits a search, while all other icons in the sidebar are safe to click~~
- [x] **Accessibility** – Scrollbar isn’t wide enough to be clicked on
- Later / TBD:
- - [x] **Accessibility** – Focus order is incorrect on the mobile version (it should be possible to move to the sidebar after having toggled it)

**Comment by phildexter** [comment]
**Must**

> Visual design – Change “Edit your account” to “Account” tooltip
>  Visual design – Move the bird higher up and slightly smaller, as per the designs
>  Visual design – Get the labels and icons to align vertically (space between icon and label)

**Should**

> Visual design – Change the tooltip to the darker indigo
> Visual design – Expand the space between the menu items (add 3px padding) - then adjust the flyout menu items spacing too
>  Visual design – Make the arrows slightly smaller and closer to the icons on slim view
>  Visual design – Slim side bar on bird hover, make the bird a little big bigger (don’t make it shrink as much)
>  Animations – The avatar suddenly jumps to the right when you collapse the menu

**Could**

>  Visual design – When the sidebar loads, the area underneath is grey. Could we switch this to match the darker hue of the sidebar’s background?
>  Animations – The close animation for sub-menus doesn't seem to play when the menu is expanded
>  Animations – The Bird seems to have two hover states (try slowly moving your mouse cursor from top to bottom and you'll see it's wing appears before the hover animation is triggered). Not sure if this is intended behaviour.

**Other**
This seems fine to me to leave as is
>  Animations – I wonder if the new |-> icon should flip horizontally rather than roll? (or maybe not animate at all?) The rolling animation feels a little strange as I think the pipe is meant to represent the always vertical menu

Can't reproduce this
 > Animations – The account menu seems to have a different tween animation to the rest of the menu. Causing it to do a weird thing when you collapse the menu while the account menu is open

These seem conflicting to me?
> Visual design – Collapsed sidebar is wider than intended
> Visual design – Make the width 60px
>  Accessibility – Scrollbar isn’t wide enough to be clicked on




Not sure I understand what is wrong with this - the UX seems fine here to me?

>  UX – Unexpected behavior with the search – clicking the "Search" icon submits a search, while all other icons in the sidebar are safe to click

**Comment by thibaudcolas** [comment]
Also noting for consideration:

- #8063
- #8055

**Comment by acnologia** [comment]
I will take a look at some/most of these @thibaudcolas

**Comment by thibaudcolas** [comment]
@acnologia this isn’t up for grabs for the time being, I believe we want @stevedya working on this. I’ll update here if we are happy for some of the above to be worked on by others.

**Comment by acnologia** [comment]
oh okay, no problem ... are there any other front-end issues that core isn't working on and need fixing?
